### PR TITLE
add perf env

### PR DESCRIPTION
**Pull request by AngryBerryMS**

### CODE REVIEW

File: src/docfx/config/ops/OpsAccessor.cs [hunk 1]
```diff
@@ -253,7 +254,9 @@ private static string BuildApi(DocsEnvironment? environment = null)
             return (environment ?? DocsEnvironment) switch
             {
                 DocsEnvironment.Prod => "https://buildapi.docs.microsoft.com",
-                _ => "https://BuildApiPubDev.azurefd.net",
+                DocsEnvironment.PPE => "https://BuildApiPubDev.azurefd.net",
+                DocsEnvironment.Perf => "https://op-build-test.azurewebsites.net",
+                _ => throw new NotSupportedException(),
```
Comment (live1206) [inline src/docfx/config/ops/OpsAccessor.cs:259]: throw `InvalidOperationException` like the others

File: src/docfx/config/ops/OpsAccessor.cs [hunk 2]
```diff
@@ -232,6 +232,7 @@ private static HttpMiddleware BuildMiddleware(DocsEnvironment? environment = nul
                         {
                             DocsEnvironment.Prod => s_opsTokenPublic,
                             DocsEnvironment.PPE => s_opsTokenPubDev,
+                            DocsEnvironment.Perf => s_opsTokenPubDev,
```
Comment (live1206) [inline src/docfx/config/ops/OpsAccessor.cs:235]: Does this need to be a separate source environment?

Comment (AngryBerryMS) [reply src/docfx/config/ops/OpsAccessor.cs]: Updated.

File: src/docfx/config/ops/OpsAccessor.cs (hunk 1 above)
Comment (AngryBerryMS) [reply src/docfx/config/ops/OpsAccessor.cs]: Done.

**Review by live1206 — APPROVED** [review]

**Pull request merged**